README.fr.md

```diff
@@ -2808,6 +2809,41 @@ Avec les options :
 -x # active le mutex lock (off par défaut)
 ```
 
+### Multi-threading
+
+_Cette partie est basée sur [une réponse StackOverflow][so-answer] par Konstantin._
+
+Sinatra n'impose pas de modèle de la concurrence. Sinatra est thread-safe, vous pouvez
+donc utiliser n'importe quel gestionnaire Rack, comme Thin, Puma ou WEBrick en mode
+multi-threaded.
+
+Lorsque le serveur est démarré, vous devez spécifier l'appel de méthode correcte pour
+chaque gestionnaire Rack particulier.
```

Comment:
I would change the 2 lines above to something like "Cela signifie néanmoins qu'il vous faudra spécifier les paramètres correspondant au gestionnaire Rack utilisé lors du démarrage du serveur."